libmamba/src/core/activation.cpp

```diff
@@ -1215,4 +1215,97 @@ namespace mamba
 
         return out.str();
     }
+
+
+    std::string NuActivator::shell_extension()
+    {
+        return ".nu";
+    }
+
+    std::string NuActivator::shell()
+    {
+        return "nu";
+    }
+
+    std::string NuActivator::hook_preamble()
+    {
+        return "";
+    }
+
+    std::string NuActivator::hook_postamble()
+    {
+        return "";
+    }
+
+    fs::u8path NuActivator::hook_source_path()
+    {
+        return Context::instance().prefix_params.root_prefix / "etc" / "profile.d" / "micromamba.nu";
+    }
+
+    std::pair<std::string, std::string>
+    NuActivator::update_prompt(const std::string& /*conda_prompt_modifier*/)
+    {
+        return { "", "" };
+    }
+
+    std::string NuActivator::script(const EnvironmentTransform& env_transform)
+    {
+        std::stringstream out;
+        auto convert_to_nulist = [](std::string command)
+        {
+            /*Checks shell variable being set and converts shell lists to nushell lists if
```

Comment:
I still have no clue what this function does.

The name indicates it converts something (what?) to a nu list.
The signature indicates it converts a command to a nu list.
The caller indicates it converts an environment variable value to a nu list.

What does it mean to "check shell variable being set"? Are you talking about `$SHELL` here? If so I don't find any reference to `$SHELL` in the function body. Are you talking about something else?

Reply:
In nushell the path is indexed (see below). It is not a singular string like in bash. The function converts @evar to a list if it detects the `:` delimiter. That's it. 
![image](https://github.com/mamba-org/mamba/assets/19485143/1ccd2506-f1a2-47ff-8b62-a5bfeec0be2d)

Reply:
There is, however, an edge case that this does not consider. Namely if for some reason the `evar` being set contains a path with a colon. I am open to suggestion to handle this differently.